src/style.css

```
@@ -1 +1,103 @@
 /* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.header {
+  width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
+  margin-bottom: 74px;
+}
+
+.header-2 {
+  width: 1024px;
+}
+
+.homebutton {
```

use camelCase or separators
```suggestion
.homeButton {
```

```suggestion
.home__button {
```